phnt/include/ntldr.h

```diff
@@ -677,19 +677,11 @@ typedef struct _PS_SYSTEM_DLL_INIT_BLOCK
     ULONG_PTR ScpArm64EcCfgCheckESFunction;
 } PS_SYSTEM_DLL_INIT_BLOCK, *PPS_SYSTEM_DLL_INIT_BLOCK;
 
-// rev
-#if (PHNT_VERSION >= PHNT_WINDOWS_10)
+#if (PHNT_VERSION >= PHNT_WINDOWS_8)
```

Comment:
Microsoft define the type in the Windows SDK 10.0.10240.0/um/minwin/wow64t.h

```
#if defined(_NTDLLBUILD_)
extern PS_SYSTEM_DLL_INIT_BLOCK LdrSystemDllInitBlock;
#else
extern NTSYSAPI PS_SYSTEM_DLL_INIT_BLOCK LdrSystemDllInitBlock;
#endif // defined(_NTDLLBUILD_)
```

Reply:
Interesting, didn't know it was there. Either way, the export and corresponding type exists in ntdll and its symbols all the way since Windows 8 RTM.